lib/v2/screens/explore_screens/vote/interactor/viewmodels/vote_state.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:equatable/equatable.dart';
+import 'package:seeds/v2/datasource/remote/model/proposals_model.dart';
+import 'package:seeds/v2/domain-shared/page_state.dart';
+
+/// --- STATE
+class VoteState extends Equatable {
+  final PageState pageState;
+  final Map<String, Map> proposalTypes;
+  final Map<String, String>? currentType;
```

Comment:
should current type be not nullable and be initialized to a type?

Reply:
It takes a value as soon as the tab is moved, I could put a default one I see no problem.